packages/flutter_tools/lib/src/emulator.dart

```diff
@@ -27,21 +28,24 @@ class EmulatorManager {
     required ProcessManager processManager,
     required AndroidWorkflow androidWorkflow,
     required FileSystem fileSystem,
-  }) : _androidSdk = androidSdk,
-       _processUtils = ProcessUtils(logger: logger, processManager: processManager),
-       _androidEmulators = AndroidEmulators(
-        androidSdk: androidSdk,
-        logger: logger,
-        processManager: processManager,
-        fileSystem: fileSystem,
-        androidWorkflow: androidWorkflow
-      ) {
+    required OperatingSystemUtils operatingSystemUtils,
+  })  : _androidSdk = androidSdk,
+        _processUtils =
```

Comment:
Here and elsewhere, you seem to have added an extra level of indentation to multi-line chunks of code. Although it isn't wrong, to remain consistent with the rest of the codebase, let's stick with a single level of indentation for these lists.

Reply:
It also makes it harder to tell what you changed vs what is an indentation update, as well as increase the chance of a merge conflict (this PR is currently conflicted)